appsettings.override.json

```diff
@@ -0,0 +1,153 @@
+    "JwtLifetimeInSeconds": 300,
+    "ApiKey": "test"
+  },
+  "Cors": {
+    "AllowedOrigins": "http://localhost:8080;https://localhost:8080",
+    "ExposedHeaders": ""
+  },
+  "SwaggerUi": {
+    "TokenUrl": "http://localhost:5000/connect/token",
+    "Enabled": true
+  },
+  "Infrastructure": {
+    "EventBus": {
+      "Vendor": "RabbitMQ", // possible values: InMemory, RabbitMQ, GoogleCloud, Azure
+      "ConnectionInfo": "localhost",
+
+      "RabbitMQUsername": "guest", // only available for RabbitMQ
+      "RabbitMQPassword": "guest", // only available for RabbitMQ
+      "ConnectionRetryCount": 5, // only available for RabbitMQ
+
+      "GcpPubSubProjectId": "", // only available for Google Cloud Pub/Sub
+      "GcpPubSubTopicName": "" // only available for Google Cloud Pub/Sub
+    },
+
+    //Database migrator
```

Comment:
It's also for the admin ui, afaik. I don't think we need this comment.

Reply:
The one I commented on (=> `//Database migrator`)

Reply:
Ok